### PR TITLE
Component/generate embeddings updated

### PR DESCRIPTION
**Pull request by tillwenke**
knowing that dict args are supported now

### CODE REVIEW

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -66,4 +66,4 @@ repos:
   #       language: python
   #       entry: python scripts/component_readme/generate_readme.py
   #       files: ^components/.*/fondant_component.yaml
-  #       additional_dependencies: ["fondant"]
\ No newline at end of file
+  #       additional_dependencies: ["fondant[aws,azure,gcp]@git+https://github.com/ml6team/fondant@main"]
```
Comment (mrchtr) [inline .pre-commit-config.yaml:69]: Can you exclude this file from the PR?

**Review by mrchtr — COMMENTED** [review]
Thanks @tillwenke! Could you exclude the changes of the .pre-commit-config.yaml?

File: .pre-commit-config.yaml (hunk 1 above)
Comment (tillwenke) [reply .pre-commit-config.yaml:69]: Sure. Keep in mind. Without this I wasnt able to creat READMEs that passed the ci/cd pipeline. I needed the latest fondant version.

**Comment by PhilippeMoussalli** [comment]
Hi @tillwenke, 

Could you please resolve the merge conflicts so that we can merge the PR?

**Comment by mrchtr** [comment]
Closing this one due to outdated changes.

**Pull request closed without merging**